capstone/src/main/webapp/script.js

```diff
@@ -287,44 +290,55 @@ async function getInputFilters() {
 
   // do manual search if filter is selected, else do Nearby Search w Places API
   if (_showSmallBusiness || _showBlackOwnedBusiness) {
```

Comment:
Do _showSmallBusiness and _showBlackOwnedBusiness need to be globals? Instead, they could be local variables, and manualSearch() could take a param specifying the search type (black owned or small business)

Reply:
that's a good point, will change!

Reply:
this also reminds me that we still have unnecessary code in our regular search from our v1 design idea lol